gnomad_qc/v5/data_ingestion/federated_validity_checks.py

```diff
@@ -194,6 +208,9 @@ def validate_federated_data(
     freq_sort_order: List[str] = ["gen_anc", "sex", "group"],
     nhomalt_metric: str = "nhomalt",
     verbose: bool = False,
+    subsets: List[str] = None,
+    variant_filter_field: str = "RF",
```

Comment:
basic question, why is the default RF rather than AS_VQSR?

Reply:
not entirely sure, but left it to be consistent with previous versions